### ISSUE TITLE
Extension not working. 

### ISSUE THREAD

**Issue by nomanall** [Ademking/fb-profile-picture-viewer#1]
I tried this extension as well as other simple tricks like
```
https://graph.facebook.com/userID/picture?type=large&redirect=true&width=2000&height=2000
&
https://graph.facebook.com/USERNAME/picture?width=2000.
```

They are not working as well. Kindly take a look.

**Comment by Ademking** [comment]
@nomanall 
Version 1.0.0 is not working.. i've fixed the extension (version 1.0.1)
Waiting for review...
![image](https://user-images.githubusercontent.com/12462188/97378168-09149d00-18c2-11eb-95c3-81b3b65ca314.png)

**Comment by Ademking** [comment]
Update! It's now working
